KenticoCloud.Delivery/ResiliencePolicy/IResiliencePolicyProvider.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Net.Http;
+using Polly;
+
+namespace KenticoCloud.Delivery.ResiliencePolicy
+{
+    /// <summary>
+    /// Provides a resilience policy.
+    /// </summary>
+    public interface IResiliencePolicyProvider
```

Comment:
i guess this is to make it more future-proof in case we want to add more policies, etc., right? I mean we could have injected `IAsyncPolicy<HttpResponseMessage>` directly in the `DeliveryClient` but this makes it more flexible. is that the idea?

Reply:
Not really. The thing is that not only the `IAsyncPolicy<HttpResponseMessage>` object is required to use Polly. There's always some other wiring involved. Hence that wrapping provider.